app/src/components/Header/Header.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../shared/variables";
+.header {
+  background: url('./img/bg.jpg');
+
+  h1 {
```

Comment:
Avoid using a selector with an element, it does not make sense for CSS Modules. If you want to get all advantages of CSS Modules use classes and do style more independent. H1 depends on a parent element, it's not right for CSS Modules. Keep styles simple.

Reply:
Fixed